### PR TITLE
fix(web-nuxt): error when replying with no email identity

### LINKED ISSUE #372: when replying to convo, cant select reply from address
occasionally when replying to a convo
the field to select the sending from address errors and is not visible
image

when replying, an error is shown with "email identity not valid"

we can replicated this by rapidly switching between multiple email convos till it happens

### PR DESCRIPTION
**Pull request by McPizza0**
fixes #372

### CODE REVIEW

**Comment by McPizza0** [comment]
* **#434** <a href="https://app.graphite.dev/github/pr/un/inbox/434?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @McPizza0 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/web-app/pages/[orgShortCode]/convo/[convoId].vue [hunk 1]
```diff
@@ -530,14 +545,32 @@
...
+            </span>
+            <span>
+              convoHasContactParticipants:
+              <span class="font-mono">{{ convoHasContactParticipants }}</span>
+            </span>
+            <span>
+              orgEmailIdentities:
+              <span class="font-mono">{{ orgEmailIdentities }}</span>
+            </span>
+          </div>
```
Comment (BlankParticle) [inline apps/web-app/pages/[orgShortCode]/convo/[convoId].vue:569]: remove this debugging stuff

**Review by BlankParticle — CHANGES_REQUESTED** [review]

File: apps/web-app/pages/[orgShortCode]/convo/[convoId].vue (hunk 1 above)
Comment (McPizza0) [reply apps/web-app/pages/[orgShortCode]/convo/[convoId].vue]: good catch 🥷

**Review by BlankParticle — APPROVED** [review]

**Comment by McPizza0** [comment]
### Merge activity

* **May 7, 1:24 PM EDT**: [Graphite](https://app.graphite.dev/github/pr/un/inbox/434) rebased this pull request as part of a merge.

**Pull request merged**